tests/integration/harvest/test_transform.py

```diff
@@ -46,9 +46,9 @@ def test_invalid_transform_iso19115_2(
 
         job_errors = interface.get_harvest_record_errors_by_job(harvest_job.id)
         assert len(job_errors) == 1
-        assert job_errors[0].message == expected_error_msg
+        assert job_errors[0][0].message == expected_error_msg
```

Comment:
we have an array inside an array. what does that represent?

Reply:
same reason like above, so the first item contains all fields from harvestRecordError, which need to be referenced using two-dimensional indexing.

Reply:
fixed.